src/anaconda_auth/repo.py

```diff
@@ -257,18 +257,63 @@ def configure_conda(
 
 
 @app.command(name="uninstall")
-def uninstall_token(org_name: str = typer.Option("", "-o", "--org")) -> None:
+def uninstall_token(
+    org_name: str = typer.Option("", "-o", "--org"),  # type: ignore
```

Comment:
Why does this need to be ignored

Reply:
Tested this locally. You shouldn't need the ignore.
```suggestion
    org_name: str = typer.Option("", "-o", "--org"),
```